### PR TITLE
Improve mathjax integration

### LINKED ISSUE #933: [liquid_tags] Inline math is not rendered with MathJax
`liquid_tags` fails to render inline math for me. I managed to have inline math rendered properly when using "plain" markdown. See for example [this example](http://drorata.github.io/posts/2017/Jan/13/first-post/) with sources available [here](https://github.com/drorata/drorata.github.io/blob/source/content/first-post.md).

However, inline math in a Jupyter notebook is not rendered properly even though it does look OK inside the Jupyter environment. An example can be found in the following places:

- [The rendered post](http://drorata.github.io/posts/2017/Mar/07/minimal-example-with-linear-regression/)
- [Markdown source](https://github.com/drorata/drorata.github.io/blob/source/content/linear-regression-minimal-example.md)
- [The notebook's source](https://github.com/drorata/drorata.github.io/blob/source/content/notebooks/linear-regression-minimal-example.ipynb)

As you can see, Github renders the inline math properly, but when included in a post, this breaks. I failed to understand where the problem is. Any help would be appreciated.

**Update 1:**
It seems like the function `init_mathjax()` is not executed when the `HTML` page is loaded. If I run it manually from the page's console, the inline math is rendered properly.

**Update 2:** I cross posted the issue also on [SO](https://stackoverflow.com/questions/46026938/inline-math-rendering-when-using-pelican-and-liquid-tags)

### PR DESCRIPTION
**Pull request by drorata**
Closes #933

### CODE REVIEW

**Comment by drorata** [comment]
@isms Who could approve this PR? Or comment if needed :)

**Comment by isms** [comment]
Hi again @justinmayer - just looping you in on this PR when you get a chance. Thanks :+1:

**Comment by justinmayer** [comment]
Any comments from @barrysteyn or other @getpelican/reviewers?

**Comment by barrysteyn** [comment]
I am sorry Justin, I know I have been slow. I will comment before the
weekend is over, I promise.


Barry Steyn
Blog <http://doctrina.org> LinkedIn <http://ca.linkedin.com/in/barrysteyn>
GitHub <https://github.com/barrysteyn>

**Comment by drorata** [comment]
Is there anything I can do to help with closing this PR? :)

**Comment by justinmayer** [comment]
@drorata: Would you please [rebase & squash your commits](https://github.com/getpelican/pelican/wiki/Git-Tips#squashing-commits)?

**Comment by drorata** [comment]
@justinmayer I hope the recent update meets the expectations.

File: Readme.rst [hunk 1]
```diff
@@ -84,8 +84,6 @@ Events                    Add event start, duration, and location info to post m
 
 Extract table of content  Extracts table of contents (ToC) from ``article.content``
 
-Feed Summary              Allows article summaries to be used in ATOM and RSS feeds instead of the entire article
-
```
Comment (justinmayer) [inline Readme.rst]: This ReadMe change is from a commit that has already landed in master and should not be present in this pull request.

**Comment by justinmayer** [comment]
@drorata: Unfortunately, I'm not sure this was rebased properly. This PR is showing changes to the ReadMe that are already in master and don't belong in this PR.

**Comment by drorata** [comment]
@justinmayer I don't know how it happened. I will try to fix it.

**Pull request closed without merging**